portal/templates/cluster_node_details.html

```diff
@@ -0,0 +1,90 @@
+{%extends "base.html"%}
+
+{%block title%}
+  Node Details
+{%endblock%}
+
+{%block body%}
+<section id="cluster-profile" class="bg-light">
+  <div class="container col-lg-12 mx-auto">
+
+    <div class="row">
+      <div class="col-lg-12 mx-auto">
+        <ol class="breadcrumb">
+          <li class="breadcrumb-item"><a href="{{url_for('dashboard')}}">Home</a></li>
+          <li class="breadcrumb-item"><a href="{{url_for('list_clusters')}}">Clusters</a></li>
+          <li class="breadcrumb-item"><a :href="'/clusters/'+cluster_name">{[{cluster_name}]}</a></li>
+          <li class="breadcrumb-item active">{[{node_name}]}</li>
```

Comment:
For {[{ cluster_name }]} and {[{ node_name }]}, you're passing `cluster_name` and `node_name` from python, and so these can be called in the Flask jinja template simply with {{ cluster_name }} and {{ node_name }}. You only want to use the delimiter {[{ some_variable }]} when the variable is generated by the VueJS (explained in comment below)